docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -0,0 +1,79 @@
+---
+description: To prevent "resource not accessible by integration" error when running job to upload findings to GitHub's Advanced Security Dashboard
+tags:
+  - Semgrep CI
+  - GitHub
+---
+
+## Why aren't findings populating in the GitHub Advanced Security Dashboard after running Semgrep in CI?
+
+When scanning with Semgrep in CI, findings automatically populate in the Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, you must first run an alternate job that uploads findings in the form of a `SARIF` file to the dashboard. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
+
+Once you run the alternate job, the job may fail and return a "resource not accessible by integration" error which has two possible causes:
+
+**1. Your repo is private**
+
+Third-party code scanning findings can only populate in the Advanced Security Dashboard if the repo is public/open source. Otherwise, the job will not have sufficient permissions to write to the dashboard regardless of any permissions set at the workflow or job level.
```

Comment:
I would just delete open source here provided public is the relevant GH distinction.